all-in-one/README.md

```diff
@@ -1,6 +1,6 @@
 # am-all-in-one
 
-![Version: 4.2.0-1](https://img.shields.io/badge/Version-4.2.0--1-informational?style=flat-square) ![AppVersion: 4.2.0](https://img.shields.io/badge/AppVersion-4.2.0-informational?style=flat-square)
+![Version: 4.3.0](https://img.shields.io/badge/Version-4.3.0-informational?style=flat-square) ![AppVersion: 4.3.0](https://img.shields.io/badge/AppVersion-4.3.0-informational?style=flat-square)
```

Comment:
Shall we use the version as 4.3.0-1. We need to maintain the Helm Chart version for a specific APIM version as well.